docs/Q-and-A.md

```diff
@@ -0,0 +1,746 @@
+{TBW}
+## How to delegate control over my private keys that control my identifiers?
+{TBW}
+# Q&A section Blockchain
+
+## what does an ID system gain from a blockchain?
+I would actually harken back to what Satoshi first called Bitcoin before it was blockchain, which was timechain. It turns out that `PKI`, which is public key infrastructure and that's like the thing that backs identifiers. I'll give you an analogy of what one is today. An existing example is the DNS system.
+
+The identifier is a domain name and then the backing PKI data is your zone file in DNS that lists your named servers, routing information, then there's the certificate of authorities that have the sort of the backing cryptography. So that's an example of a PKI system. Every PKI system, if it isn't centralized, like a system where they just kind of decree that this users is usurped, if you want to decentralize that, the most important piece is having a global immutable append-only log, because everyone needs to see the state of some identifier. Like if Alice creates an ID and she initially associates, say her phone that she has at the time with her ID and a public key on it, when she goes to get a new phone, she's going to switch that key up.
+
+So everyone needs to be able to see that event, globally and deterministically, or else someone could masquerade as Alice. They could say, "Well, I have her old key so I'm her." You have to have this sort of linear chronology of what happened when, it's a state machine. So what a blockchain does, really elegantly that we never had before, is it's an oracle for entering events into a chronology. That's exactly what we use it for. In its most primitive sense, we use Bitcoin to enter in events in a chronology that nodes sort of watch for and compute so they all can understand the state of every ID that's anchored in Bitcoin. So it's very important.\
+[Source: What bitocin Did 231](https://www.whatbitcoindid.com/wbd231-daniel-buchner)
+
+## So why would we pick Bitcoin versus some other blockchain, like Ethereum or Hyperledger?
+Because we're not actually concerned with any of the super sexy smart contracts, interesting... Whatever you might call it. Not to say that there's less interesting stuff in Bitcoin, but we just don't care about features like that, that is not applicable for us. What we most care about is entering stuff in the most immutable, hardened record imaginable and right now, that's Bitcoin. It's pretty hard to argue that isn't Bitcoin, so that's why we use it, is because it has those attributes and it suffices, it does exactly what we need it to do.\
+[Source: What bitocin Did 231](https://www.whatbitcoindid.com/wbd231-daniel-buchner)
+
+## Does Sidetree use a blockchain?
+Yes, it needs a global blockchain to provide an ordered settlement layer 1 that has reached consensus over the unique representation of the "truth". Sidetree sits on top of this Layer 1, in layer 2 of the `ToIP model` and is blockchain agnostic.\
```

Comment:
Usage of a "global blockchain" is an implementation choice for a DID method - it is not an inherent property of Sidetree.

Reply:
Thx Troy, I copied the wording from https://www.whatbitcoindid.com/wbd231-daniel-buchner so could you elaborate on how to change the text that it fits the current state?

Reply:
I would say instead:
Not exactly-- a Sidetree based on KERI or other consensus mechanisms and roots-of-trust would, in theory, be possible (if not exactly efficient!), and blockchain-like data structures such as Amazon's QLDB would also work.

Reply:
I've tried to find best of both worlds: 
Yes, it could use a global blockchain to provide an ordered settlement layer 1 that has reached consensus over the unique representation of the "truth". Sidetree sits on top of this Layer 1, in layer 2 of the `ToIP model` and is blockchain agnostic.\
A Sidetree based on other consensus mechanisms and roots-of-trust would, in theory, be possible (if not exactly efficient!), and blockchain-like data structures such as Amazon's QLDB would also work.